playbooks/roles/tinyproxy/templates/tinyproxy.service.j2

```diff
@@ -6,8 +6,8 @@ Documentation=https://www.banu.com/tinyproxy/
 
 [Service]
 Type=forking
-PIDFile=/var/run/tinyproxy/tinyproxy.pid
-ExecStart=/usr/sbin/tinyproxy -c /etc/tinyproxy.conf
+PIDFile={{ tinyproxy_pid_file }}
+ExecStart={{ tinyproxy_startup_script }} {{ tinyproxy_conf_file }}
 ExecStop=/usr/bin/killall -9 tinyproxy
```

Comment:
I think we can get along without an `ExecStop` at all, now that I read [systemd.kill](https://www.freedesktop.org/software/systemd/man/systemd.kill.html). It might be worthwhile as an `ExecStopPost`.